### ISSUE TITLE
Asking here because didn't find another place

### ISSUE THREAD

**Issue by omgwtfbbqpls** [nostrenz/cshap-discord-rpc-demo#2]
What are the sources of the music in the JanusVR Touhou rooms? You should consider adding them somewhere.

**Comment by nostrenz** [comment]
True, some of them are from the original games but others are arrangements I probably got from Youtube.
Anyway I'll try to find the correct sources for each track and update the readme.
